lib/onigumo/parser.ex

```diff
@@ -7,7 +7,6 @@ defmodule Onigumo.Parser do
     root_path
     |> File.ls!()
 	|> Enum.filter(fn filename -> Path.extname(filename) == ".raw" end)
-    |> Enum.join("\n")
-    |> IO.puts()
+    |> Enum.map(&IO.puts(&1))
```

Comment:
Interesting indentation. Is this indentation OK?

We can also use similar approach to python:
```
iex(4)> Path.absname("foo.raw") |> String.ends_with?(".raw")
true
```

Reply:
The indentation is not OK. I’ll fix it.